src/rpmfusion-cert.py

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
Please verify that it will work on EL7 which still default to python2

Reply:
The code is still python2 compatible. EL7's spec file will have to edit the shebang lines, that's all. (Which it was hopefully doing anyway, replacing the previous `/usr/bin/python` with `/usr/bin/python2`.)

Reply:
I would leave it with python and to rely on the fedora normalization script to have it changed to python3 while keeping python for el7 (I don't think we need to support older).

Reply:
and not update package in el7 ? or el7 use python 3.4 or 3.6 ?

Reply:
> and not update package in el7 ?

That's actually a good point — _so far_, there's really no reason to even build this version for el7. If we fix the `--verify-cert` CRL bug, that's maybe another story.

I tested the `@PYTHON@` thing, that also works well, better than I expected actually. The `configure` script detects the path to the available python executable and creates `rpmfusion-packager-setup.py` and `rpmfusion-cert.py` files with a shebang line to match. 

It does pull in the _newest_ python available, though, so if any `python3.x` is also available on el7 it'll end up trying to use that version. May be a problem if any of the dependencies don't have python3- versions there.

Reply:
(Oh, I should've mentioned though: It's also possible to set the `$PYTHON` environment variable before calling `configure`, to force a python path. So el7 could build with `PYTHON=/usr/bin/python2 ./configure` or whatever, to bypass the detection. It'll still write the shebangs to match whatever we set `$PYTHON` to.)

Reply:
Note that there is no "fedora normalization script to have it changed to python3".

Reply:
I pushed further commits to add `--with-python2` and `--with-python3` to the configure script, along with the generation from `.py.in` files that I mentioned previously. The spec file included in the repo will use `%configure --with-python3` on Fedora and `%configure --with-python2` otherwise (el7), to generate scripts with the correct shebangs. So I think we're good on this point.

(Though I'm still toying with the idea of converting to `setup.py` and launching autotools into the sun.)